spec/features/02_visitor_sees_show_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
```

Comment:
for the file name of these capybara specs, remove the `02_` portion. We only did that in the past so that you would run tests in a specific order for marathons and SCs, but in actual releases we dont want to number them

Reply:
removed numbers at the front of the file name